source/features/mark-unread.tsx

```diff
@@ -34,7 +34,8 @@ const stateIcons = {
 	issue: {
 		open: icons.openIssue,
 		closed: icons.closedIssue,
-		merged: icons.closedIssue // Required just for TypeScript
+		merged: icons.closedIssue, // Required just for TypeScript
```

Comment:
I'm surprised as to why this TS didn't shout about this before! 🤔

Reply:
That _is_ the solution. TS is saying "I expect `issue` to have `'open' | 'merged' | 'closed' | 'draft'` properties, because later we use `icons[type][state]` which, as far as TS knows, they can be `icons.open.draft`